src/flavor/object-flavor.cpp

```diff
@@ -92,18 +92,19 @@ static bool object_easy_know(int i)
 
 /*!
  * @brief 各種語彙からランダムな名前を作成する / Create a name from random parts.
- * @param out_string 作成した名を保管する参照ポインタ
+ * @return std::string 作成した名前で
  * @details 日本語の場合 aname_j.txt 英語の場合確率に応じて
  * syllables 配列と elvish.txt を組み合わせる。\n
  */
-void get_table_name_aux(char *out_string)
+std::string get_table_name_aux(void)
```

Comment:
In C++, it is common to define a function with no arguments as `f()` instead of `f(void)`.

Reply:
The forced push should have dropped the void from the changed prototypes and function declarations.